released/packages/coq-mi-cho-coq/coq-mi-cho-coq.1.0.0/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "A specification of Michelson in Coq to prove properties about smart contracts in Tezos"
+maintainer: "raphael.cauderlier@nomadic-labs.com"
+authors: [ "Raphaël Cauderlier" "Bruno Bernardo" "Julien Tesson" "Arvid Jakobsson" ]
+
+homepage: "https://gitlab.com/nomadic-labs/mi-cho-coq/"
+dev-repo: "git+https://gitlab.com/nomadic-labs/mi-cho-coq/"
+bug-reports: "https://gitlab.com/nomadic-labs/mi-cho-coq/issues"
+license: "MIT"
+
+build: [
+  ["./configure" "michocoq" "michocott"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  make "install"
+]
+depends: [
+  "coq-list-string"
+  "coq-menhirlib" {>= "20190626"}
+  "coq-moment" {>= "1.2.0"}
+  "coq-ott" {>= "0.29"}
+  "coq" {>= "8.10" & < "8.12.0" }
+  "menhir"
+  "ocaml" {>= "4.07.1"}
+  "ocamlbuild"
+  "ott" {build & >= "0.29"}
+  "zarith"
+]
+build-test: [ make "test" ]
+
+description: """
+Michelson is a language for writing smart contracts on the Tezos blockchain.
+This package provides a Coq encoding of the syntax and semantics of Michelson,
+automatically generated by the Ott tool. Also included is a framework called Mi-Cho-Coq
+for reasoning about Michelson programs in Coq using a weakest precondition calculus."""
+tags: [
+  "category:Programming Languages/Formal Definitions and Theory"
```

Comment:
The actual category starts with `Computer Science/`.

Also, please add a tag `"date:YYYY-MM-DD"` with the release date of the library.

Reply:
Thank you, I have done both changes.